### ISSUE TITLE
add a "Operators" section to the Operator Framework SIG page

### ISSUE THREAD

**Issue by dmueller2001** [openshift-cs/commons.openshift.org#777]
@wgordon17 @jiri-fiala 

update requested  to  https://commons.openshift.org/sig/OpenshiftOperators.html

create and add a section like the users section on the okd.io page mocked up and annotated here:

![community operators section](https://user-images.githubusercontent.com/4029488/44230311-a951d180-a14f-11e8-8803-659533a2843d.png)

(a) it should be driven by a .yml file in /data directory of /commons.openshift.org repo

(b) image should be linked and clickable to a different url from the text which may be linked to a different url..for example the logo may be ‘kafka’ and there may be multiple ‘kafka’ operators but they may be built by different companies..  

(c) a 2nd listing page should be created and named simply "Operators.html" and this page should like to a page similiar to the commons.openshift.org/particpants.html page with a searchable alphabetic listing of all the operators 

see this pdf for location on the landing page for new section:
[screencapture-commons-openshift-org-sig-OpenshiftOperators-html-2018-08-16-12_12_03.pdf](https://github.com/openshift-cs/commons.openshift.org/files/2294941/screencapture-commons-openshift-org-sig-OpenshiftOperators-html-2018-08-16-12_12_03.pdf)

**Comment by jiri-fiala** [comment]
Hi @dmueller2001,

[here](https://com-stage-pro.b9ad.pro-us-east-1.openshiftapps.com/sig/OpenshiftOperators.html) you can find a preview of the requested carousel and [the operator list](https://com-stage-pro.b9ad.pro-us-east-1.openshiftapps.com/sig/operators.html) (searchable by title).
Vetted operators are to be specified in a new yaml file as follows:
```
  - title: "operator title 1"
    link: "http://www.openshift.com/"
    logo: "operators/remove.png" (link to an operator/system logo)
    logo_link: "http://www.redhat.com/" (optional, if not present, the 'link' is used)
    description: "Operator description" (optional)
```

For previewing purposes, I have used some operators from [this list](https://github.com/operator-framework/awesome-operators), but I did not bother with actual logos.

- Please have a look on the preview and provide any copy updates (for [the *"Take a few moments..."*](https://github.com/jiri-fiala/commons.openshift.org/blob/operators-carousel/source/sig/operators.html.erb#L11) lead text, etc).
- Also, please provide a list of operators we want to publish initially with preferred logos, descriptions, etc.

Thank you.

**Comment by dmueller2001** [comment]
@jiri-fiala thanks I will review and give feedback by EOW (just returned from vacation)

**Comment by dmueller2001** [comment]
@jiri-fiala @wgordon17 where/how can I update the .yaml file for operators to add content before it goes live? in @jiri-fiala's repo?

**Comment by jiri-fiala** [comment]
@dmueller2001, I have created PR #823 that is not yet to be merged, where you can update the yams file, or you can push to the operators-carousel branch in jiri-fiala/commons.openshift.org

**Comment by dmueller2001** [comment]
@jiri-fiala I need to have push access to the operator-carousel branch in  jiri-fiala/commons.openshift.org in order to upload the image..

**Comment by dmueller2001** [comment]
@jiri-fiala I just made a pull request on jiri-fiala/commons.openshift.org with the first 8 operators to be listed..could you merge and share the results?

I am going to do a second one with the next group of community operators separately..

**Comment by jiri-fiala** [comment]
@dmueller2001, I have updated the list per your PR against my fork, amended it a bit, and included the logos that you've sent via email separately.
In the new [`operators-carousel` branch](https://github.com/openshift-cs/commons.openshift.org/commits/operators-carousel), there are still some logos missing and all of them are yet to be compressed. The [preview site](https://com-stage-pro.b9ad.pro-us-east-1.openshiftapps.com/sig/OpenshiftOperators.html) has been updated.
